ci/jenkins/test.sh

```diff
@@ -375,7 +375,7 @@ function revert_snapshot_windows {
 function  build_and_deliver_antrea_windows_and_linux_containerd_images {
     echo "====== Cleanup Antrea Installation Before Delivering Antrea Windows and Antrea Linux containerd Images ====="
     clean_antrea
-    kubectl delete -f ${WORKDIR}/antrea-windows-containerd-with-ovs.yml --ignore-not-found=true || true
+    kubectl delete -f ${WORKDIR}/antrea-windows-with-ovs.yml --ignore-not-found=true || true
```

Comment:
Could you also update `WINDOWS_YAML_SUFFIX` value? It is also used as windows yaml name in test.sh.

Reply:
Good catch, I updated it in the last commit. Please check that this is what is needed.